main.go

```diff
@@ -312,13 +312,17 @@ func main() {
 
 				s.Suffix = " Checking Red Hat Insights..."
 				s.Start()
-				isRegistered := insightsIsRegistered()
+				isRegistered, error := insightsIsRegistered()
```

Comment:
`error` is a type name. If the type system doesn't throw a syntax error for naming a variable `error`, it's certainly against convention. Conventionally, the error variable is named `err`.

Reply:
You are right. I didn't noticed that GoLand marks it as type error. It uses different convention as PyCharm.